packages/@webex/widget-cc-station-login/src/station-login/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import r2wc from '@r2wc/react-to-web-component';
+import store from '@webex/widget-provider';
+import {observer} from 'mobx-react';
+
+import StationLoginPresentational from './station-login.presentational';
+import {useStationLogin} from '../hooks';
+
+const StationLogin: React.FunctionComponent = observer(() => {
+  const {loginState, setLoginState, ccSdk, isAvailable} = store;
+  const result = useStationLogin();
+
```

Comment:
we will need to add some documentation here

Reply:
What kind of documentation are you looking for Arun? Just code comments or something like API reference?

Reply:
code comments in general

Reply:
This is just dummy code. We will add comments and documentation when we write the actual code